src/Raven.Studio/wwwroot/Content/css/pages/patch.less

```diff
@@ -362,6 +362,13 @@
     }
 }
 
+.margin-toggle {
```

Comment:
let's avoid having this - try to use standard classes like: margin, padding, margin-lg, etc. Also you created global styles in view specific style. We should put such styles nested.

Reply:
done